lib/libshare/nfs.c

```diff
@@ -212,10 +138,17 @@ nfs_enable_share_one(const char *sharepath, const char *host,
 	argv[4] = hostpath;
 	argv[5] = NULL;
 
+#if DEBUG >= 2
+	int i;
+	fprintf(stderr, "CMD: ");
+	for (i = 0; i < 5; i++)
+		fprintf(stderr, "%s ", argv[i]);
+	fprintf(stderr, "\n");
+#endif
```

Comment:
Do we still need these DEBUG blocks.  If we really need to preserve this debugging I suggest updating the code to use `dprintf()`.  Then this debugging can be enabled at run time rather than build time like this, `ZFS_DEBUG="on" zfs share ...`.